Testscripts/Linux/Linux-Test-Project-Tests.sh

```diff
@@ -45,28 +48,48 @@ install_package "${common_packages[@]}"
 case $DISTRO in
     "suse"*)
         suse_packages=(git-core db48-utils libaio-devel libattr1 \
-            libcap-progs libdb-4_8 perl-BerkeleyDB)
+            libcap-progs libdb-4_8 perl-BerkeleyDB ntp)
         install_package "${suse_packages[@]}"
         ;;
     "ubuntu"* | "debian"*)
         deb_packages=(git libaio-dev libattr1 libcap-dev keyutils \
             libdb4.8 libberkeleydb-perl expect dh-autoreconf gdb \
-            libnuma-dev quota genisoimage db-util unzip exfat-utils)
+            libnuma-dev quota genisoimage db-util unzip exfat-utils ntp)
         install_package "${deb_packages[@]}"
         ;;
     "redhat"* | "centos"* | "fedora"*)
-        rpm_packages=(git db48-utils libaio-devel libattr libcap-devel libdb)
+        install_epel
+        rpm_packages=(git db4-utils libaio-devel libattr libcap-devel libdb ntp)
         install_package "${rpm_packages[@]}"
         ;;
     *)
         LogMsg "Unknown distro $DISTRO, continuing to try for RPM installation"
         ;;
 esac
 
+# Some CPU time is assigned to set real-time scheduler and it affects all cgroup test cases.
+rt_runtime_us=$(cat /sys/fs/cgroup/cpu/user.slice/cpu.rt_runtime_us)
+if [ $rt_runtime_us -eq 0 ]; then
+    echo "950000" > /sys/fs/cgroup/cpu/user.slice/cpu.rt_runtime_us
```

Comment:
what represent of 950000?

Reply:
The values for rt_period_us and rt_runtime_us (950000us or 0.95s), this gives 0.05s to be used by non-RT tasks.

Reply:
Then, please add it in comment.